test/DPoPTests/DPoPTests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
+        <TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
@brockallen, when you review this, I want to call this out. I made this change because tests need to use IdentityServer 7 to get its changes that support the new version of wilson.

The tests are creating an IdentityServer test host and then using it to get dpop tokens. The problem is that if our tests use IdentityServer 6.3 but we update the wilson dependency in IM.OC.DPoP, IdentityServer will use that wilson version too. Once we update wilson, we need the updated version of IdentityServer (that uses JsonElements instead of Dictionary<string, object>) in order to deserialize the proof tokens. And IdentityServer 7 of course only targets net8.0. So, this forces us to drop net6.0 and net7.0 from the target frameworks of this test project.

Reply:
Thinking about this more, I want to keep all the target frameworks in the tests, and solve this with conditional compilation